### DISCUSSION TITLE
v2/zh/guide/feature/page-info

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#331] [Announcements]
# v2/zh/guide/feature/page-info

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.gitee.io/v2/zh/guide/feature/page-info.html

**Comment by Ikun-CYX** [comment]
评论测试

**Comment by zuihou** [comment]
关于 pageview 参数，
文档中说道： “当配置 Waline 作为 评论功能 的提供者时，该功能默认启用”


1. 使用 giscus 时，是否支持该参数？
2. 该参数生效后，长什么样子？

**Reply by Mister-Hope** [reply]
1. 不支持
2. 长它该长的样子，如有需求自行修改模板调试
